lib/Mojolicious/Command/Author/generate/dockerfile.pm

```diff
@@ -0,0 +1,87 @@
+package Mojolicious::Command::Author::generate::dockerfile;
+use Mojo::Base 'Mojolicious::Command';
+
+use Mojo::File qw(path);
+use Mojo::JSON qw(encode_json);
+
+has description => 'Generate "Dockerfile"';
+has usage       => sub { shift->extract_usage };
+
+sub run {
+  my $self = shift;
+  my $name = $self->app->moniker;
+  my $exe  = $ENV{MOJO_EXE} ? path($ENV{MOJO_EXE})->to_rel($self->app->home)->to_string : 'script/' . $name;
+  $self->render_to_rel_file('dockerfile', 'Dockerfile', {name => $name, cmd => encode_json [$exe, 'prefork']});
+}
+
+1;
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Command::Author::generate::dockerfile - Makefile generator command
+
+=head1 SYNOPSIS
+
+  Usage: APPLICATION generate dockerfile [OPTIONS]
+
+    mojo generate dockerfile
+
+  Options:
+    -h, --help   Show this summary of available options
+
+=head1 DESCRIPTION
+
+L<Mojolicious::Command::Author::generate::dockerfile> generates C<Dockerfile> files for applications.
+
+This is a core command, that means it is always enabled and its code a good example for learning to build new commands,
+you're welcome to fork it.
+
+See L<Mojolicious::Commands/"COMMANDS"> for a list of commands that are available by default.
```

Comment:
You need to add the command to the docs there too. And the class overview in `Mojolicious::Guides`.